### PR TITLE
AO3-4297: i18n the invite confirmation email

### PR DESCRIPTION
**Pull request by zz9pzza**
https://otwarchive.atlassian.net/browse/AO3-4297

### CODE REVIEW

**Comment by cesy** [comment]
Code all looks sensible to me. Checking for an automated test of that email, I found https://github.com/otwcode/otwarchive/blob/master/features/users/user_create.feature which covers the new user confirmation, and the activation email step tests the text that's been changed here, so we should be covered. That appears to be what the code is covering, despite the subject of the pull request - it's the "activate your account" one, not "you've been invited" one. https://github.com/otwcode/otwarchive/blob/master/features/other_a/invite_request.feature tests the invite email. So this should be good to merge.

**Comment by cesy** [comment]
Jira cross-link: https://otwarchive.atlassian.net/browse/AO3-4297

File: app/views/user_mailer/signup_notification.html.erb [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-
-  <p>There's lots of information and advice on how to use the Archive in our <%= style_link("FAQ", archive_faqs_url) %>. You'll find the latest news about site developments in our <%= style_link("Archive admin posts", root_url + "admin_posts") %>. If you need more help, run into a bug, or have questions or comments, please <%= support_link("get in touch with our Support team") %>, who are always happy to help out.</p>
-
-  <p>We hope you enjoy using the Archive.</p>
-<% end %>
\ No newline at end of file
+<p><%= t('.welcome',
+           login: style_bold(@user.login)).html_safe %></p>
+<p><%= t('.html.part1',
+           activate_account_link: style_link( t('.activate_your_account'), activate_url(id: @user.activation_code))).html_safe %></p>
```
Comment (sarken) [inline app/views/user_mailer/signup_notification.html.erb]: There's an extra space before the `t('.activate_your_account')`

File: app/views/user_mailer/signup_notification.text.erb [hunk 2]
```diff
@@ -1,11 +1,16 @@
 <% content_for :message do %>
-Welcome to the Archive of Our Own, <%= @user.login %>!
+<%= t('.welcome',
+           login: @user.login ) %>
```
Comment (sarken) [inline app/views/user_mailer/signup_notification.text.erb]: There's an extra space before the `)`

File: app/views/user_mailer/signup_notification.text.erb [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+           activate_account_url: activate_url(id: @user.activation_code)) %>
 
-Once your account is up and running, you can post your fanworks, set up email subscriptions to let you know when your favorite creators or works have updated, set preferences to customize the way the site looks and works for you, keep track of the works you've viewed on the Archive via your history, and much more.
+<%= t('.text.part2') %>
 
-There's lots of information and advice on how to use the Archive in our FAQ at <%= archive_faqs_url %>. You'll find the latest news about site developments in our Archive admin posts at <%= root_url %>admin_posts. If you need more help, run into a bug, or have questions or comments, please get in touch with our Support team, who are always happy to help out: <%= root_url %>support.
+<%= t('.text.part3', 
+           faq_url: archive_faqs_url,
+           admin_posts_url: admin_posts_url, 
+           contact_support_url: root_url + 'support') %>
```
Comment (sarken) [inline app/views/user_mailer/signup_notification.text.erb]: Can this be `new_feedback_report_url` or `support_url`?

File: config/locales/en.yml [hunk 4]
```diff
@@ -54,16 +53,41 @@ en:
...
+                  set preferences to customize the way the site looks and works for you, keep
+                  track of the works you've viewed on the Archive via your history, and much more.
+        part3: |-
+                  There's lots of information and advice on how to use the Archive in our FAQ
+                  at %{faq_url}. You'll
+                  find the latest news about site developments in our Archive admin posts at
+                  %{admin_posts_url}. If you need more
+                  help, run into a bug, or have questions or comments, please get in touch with
+                  our Support team, who are always happy to help out:
+                  %{contact_support_url} .
```
Comment (sarken) [inline config/locales/en.yml]: The space before the period should probably be removed here, since `%{faq_url}.` and `%{admin_posts_url}.` don't have them.

**Pull request merged**